common/components/TXMetaDataPanel/components/SimpleGas.tsx

```diff
@@ -112,19 +118,41 @@ class SimpleGas extends React.Component<Props> {
               <span>{translate('TX_FEE_SCALE_RIGHT')}</span>
             </div>
           </div>
-          <FeeSummary
-            gasPrice={gasPrice}
-            render={({ fee, usd }) => (
-              <span>
-                {fee} {usd && <span>/ ${usd}</span>}
-              </span>
-            )}
-          />
+          {this.renderFee()}
         </div>
       </div>
     );
   }
 
+  private renderFee() {
+    const { gasPrice, scheduling, scheduleGasPrice } = this.props;
+
+    if (scheduling) {
+      return (
+        <SchedulingFeeSummary
+          gasPrice={gasPrice}
+          scheduleGasPrice={scheduleGasPrice}
+          render={({ fee, usd }) => (
+            <span>
+              {fee}&nbsp;{usd && <span>/&nbsp;${usd}</span>}
```

Comment:
Unnecessary `$` in here (That's for string interpolation, not React variable printing.) Renders as `$ETH` now.

Reply:
Nope, it's meant to be like that. Please see the current version, it displays $ in front of value and USD after the value.